### ISSUE TITLE
Main page content is not contained in a landmark

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#300]
### Steps to reproduce
Install and run an audit with the aXe extension

### Solutions
In general all content should be contained within landmarks like `<header>`, `<main>`, etc.
[Here's a write up on what those look and how to use them.](https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html)

**Comment by carmacleod** [comment]
Recommendations:

![image](https://user-images.githubusercontent.com/3331913/48031759-3ee37880-e123-11e8-8754-fb34bc303f00.png)

- The `div` with `class="page-header"` should use `<header>` instead of `<div>`.
- The `div` with `class="page-content"` should be the `<main>` instead of `<div>`.

**Comment by carmacleod** [comment]
Try out this awesome [landmark browser extension](http://matatk.agrip.org.uk/landmarks/), which makes designing and testing your page's landmarks easy.  :)
